docs/pr-quality/colonist-take-home-test-opening-pr.md

```diff
@@ -0,0 +1,12 @@
+# Colonist Take Home Test Opening PR
+
+1. Create an empty Github repository
+1. Pull repo
+1. Checkout a new branch following proper [naming conventions](/docs/code-quality/fundamentals.md#branch-naming)
+1. Add the code that you wrote in JSFiddle to that branch. Ensure the JSFiddle link is public
+1. Push branch
+    - Ensure code abides by the developer [fundamentals](/docs/code-quality/fundamentals.md), [intermediate](/docs/code-quality/intermediate.md), and [advanced](/docs/code-quality/advanced.md) practices
+1. Create a PR that merges to master following proper [naming conventions](/docs/code-quality/fundamentals.md#pr-naming)
+    - PRs must follow a strict criteria; refer to [PR format](/docs/pr-quality/pr-format.md)
+1. Ensure repo is public
+1. Send the PR link to your contact for review
```

Comment:
Numbers

Reply:
The list will automatically number them as expected if reading in 'preview' format. Should it still use consecutive numbering?

1. First
1. Second

Reply:
yeah that would be better, unless that messes up the formatting in the preview format

Reply:
Sure will change to consecutive numbers. It doesn't mess up formatting, this is just another way of making consecutive lists I use.